locale/en/site.json

```
@@ -82,7 +82,11 @@
       "verify-text": "How to verify"
     },
     "install-on-linux": {
-      "text": "Installing Node.js via binary archive"
+      "text": "Installing Node.js via binary archive on Linux"
+    },
+    "install-on-wsl": {
+      "link": "https://docs.microsoft.com/windows/dev-environment/javascript/nodejs-on-wsl",
+      "text": "Install on Windows Subsystem for Linux (WSL)",
```

```suggestion
      "text": "Install on Windows Subsystem for Linux (WSL)"
```

No need to include the link, we can just embed it (unless we are planning to share a localized link

---

No need to localize the link. The linked doc auto-sniffs the users loc and updates the language accordingly.